File: FSWO102-CS/Lesson8/HandsOnL08/HandsOnL08/Classes/Cat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using HandsOnL08.Interfaces;

namespace HandsOnL08.Classes
{
    public class Cat : IMammal
    {
        public void Eat()
        {
            Console.WriteLine("Cats eat mice.");
        }

        public void Run()
        {
            Console.WriteLine("Cats can run at a top speed of 30 mph!");
        }

        public void Speak()
        {
            Console.WriteLine("Meow!");
        }
    }
}

File: FSWO102-JV/lesson08/handsOnL08/Cat.java
package handsOnL08;

public class Cat implements Mammal {

	@Override
	public void speak() {
		System.out.println("Meow!");

	}

	@Override
	public void run() {
		System.out.println("Cats can run at a top speed of 30 mph!");

	}

	@Override
	public void eat() {
		System.out.println("Cats eat mice.");
		
	}
	
	public static void main(String[] args) {
//		Cat mrwhiskers = new Cat();
		Mammal mrwhiskers = new Cat();
		mrwhiskers.speak();
		mrwhiskers.run();
		mrwhiskers.eat();

	}

}
